llm/ggml.go

```diff
@@ -43,6 +43,14 @@ func (kv KV) Architecture() string {
 	return "unknown"
 }
 
+func (kv KV) IsAdapter() bool {
```

Comment:
This can probably generalized into a `GeneralType` or `Kind` method instead of specifically checking for `adapter`